CONTRIBUTING.md

```diff
@@ -3,12 +3,15 @@
 Thank you for considering to contribute to Compiled!
 Pull requests, issues and comments are welcome.
 
-For pull requests,
-please:
+## Contribution Standards
 
 - Add tests for new features and bug fixes
 - Follow the existing style
 - Separate unrelated changes into multiple pull requests
+- As Compiled has not been officially released, when introducing a breaking change, please add a changeset by calling `yarn changeset` and select `minor`.
```

Comment:
Isn't a changeset required for any change, regardless of the type?
I would probably reframe this list differently. For example

A changeset is required...run ... select `minor`/`major`/`patch`.... 

Also what do we mean with " As Compiled has not been officially released"

Reply:
> Also what do we mean with " As Compiled has not been officially released"

I think it's referring to how Compiled has not yet had a stable release (v1.0)

We probably should make a stable release at some point but it's not in our priorities at the moment